src/main/java/Exercise2.java

```diff
@@ -0,0 +1,10 @@
+public class Exercise2 {
+    public int calculateSumOfDigitf(int x){
+        int Sum = 0;
```

Comment:
Variable name should be lowercase. `sum`

Reply:
Done